External/README.md

```diff
@@ -18,6 +22,29 @@ repositories enabled.
 
 The example can be deployed and run locally or on a remote Kubernetes cluster.
 
+### Local deployment
+
+The local deployment option relies on a local Kubernetes cluster and has been tested with Docker Desktop's built-in Kubernetes feature and with Minikube.
+
+User metadata is passed to the application via HTTP headers. For the local deployment, we are mocking the headers
+by injecting test user metadata into incoming requests via Istio.
+
+#### Docker Desktop
```

Comment:
do we push Minikube first as a the preferred solution or do we consider Docker Desktop is still important as providing more stability/simplicity/... ?